src/cryptonote_core/service_node_list.cpp

```diff
@@ -404,6 +522,33 @@ namespace service_nodes
     return true;
   }
 
+  static int count_bits(unsigned char byte)
+  {
+    return byte == 0 ? 0 : 1 + count_bits(byte & (byte-1));
```

Comment:
`return (byte & 1) + count_bits(byte >> 1);`

Reply:
hey, nice one

Reply:
except that it crashes...

Reply:
and mine recurses the number of set bits times, vs yours which recurses 8 times if you include your terminating case

Reply:
im a nub. I'd just do the non-recursive soln.